### PR TITLE
WRQ-7108: Remove unnecessary vendor prefixes

### PR DESCRIPTION
**Pull request by vJIYEv**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Agate has CSS vendor prefixes. Most of them are not required anymore.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Considering browser versions we support, I removed unnecessary vendor prefixes.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-7108

### Comments
Enact-DCO-1.0-Signed-off-by: Jiye Kim ([jiye.kim@lge.com](mailto:jiye.kim@lge.com))

### CODE REVIEW

**Comment by MikyungKim** [comment]
@vJIYEv , Please update your branch with the latest develop.

File: Panels/Panels.module.less [hunk 1]
```diff
@@ -85,7 +85,6 @@
 
 			.viewport {
 				&.transitioning {
-					-webkit-clip-path: polygon(@agate-panels-breadcrumb-width 0, 100% 0, 100% 100%, @agate-panels-breadcrumb-width 100%);
```
Comment (MikyungKim) [inline Panels/Panels.module.less:88]: There are 2 places more in this file in develop branch, `-webkit-margin-start` and `-webkit-margin-end`. Please fix those properties.

Comment (vJIYEv) [reply Panels/Panels.module.less:95]: OK

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by seunghoh — APPROVED** [review]

**Review by seunghoh — APPROVED** [review]

**Pull request merged**